lib/config.js

```diff
@@ -41,4 +41,41 @@ export default {
     maximum: 1.0,
     order: 5,
   },
+
+  projectDefaults: {
+    type: "object",
+    title: "Default Project Settings",
+    properties: {
+      outputPath: {
+        title: "Output Path",
+        description: "The path relative to a project root where the compilation \
+                      output will be saved.",
+        type: "string",
+        default: ".latex",
+      },
+
+      postCompileAction: {
+        title: "Post Compile Action",
+        description: "Action to be performed after the compilation. Note that, if \
+                      `link` or `copy` is chosen, any file at `Post Compile Action \
+                      Path` whose name clashes with `'texroot'.pdf` will be overwritten. \
+                      Invoke the `edit` command to chek a project `texroot`.",
+        type: "string",
+        default: "none",
```

Comment:
I set it to "none" as default because to activate it new users would be forced to go there and read that it can potentially overwrite files if set to "link" or "copy". But sure, it is probably better to keep the behaviour consistent for existing users.